.rubocop_local.yml

```diff
@@ -0,0 +1,3 @@
+AllCops:
+  Exclude:
+  - db/schema.rb
```

Comment:
Uh, does this make sense having this here?

Reply:
probably not...I must have that in my tool that way :/